### PR TITLE
Removed Unwanted Line Spacings

### LINKED ISSUE #18: Unwanted Line Spacing among Lines

### PR DESCRIPTION
**Pull request by naveentherocker**
Fixes #18 
Removed Unwanted Line Spacings in

1. .travis.yml
2. README.md
3. css/tiles.css

### CODE REVIEW

**Comment by naveentherocker** [comment]
@xeon-zolt Review this too Please

File: README.md [hunk 1]
```diff
@@ -53,4 +53,8 @@ If you like to join developing,
 - you can find/create [issues](https://github.com/fossasia/labyrinth/issues) and solve them.
   - When you solve an issue, you do not own it. Share your progress via a Pull-Requst as soon as possible.
   - Discuss with others who work on the issue about the best solution. It is your responsibility, not the maintainer's to choose the best solution.
+<<<<<<< HEAD
```
Comment (xeon-zolt) [inline README.md]: something slipped in while merging

**Comment by naveentherocker** [comment]
Fixed

**Comment by naveentherocker** [comment]
What about Merging it @xeon-zolt

**Comment by xeon-zolt** [comment]
let others approve it too

**Comment by naveentherocker** [comment]
I didn't get you @0x48piraj

**Comment by niccokunzmann** [comment]
@isuruAb @0x48piraj You can click "Squash and merge". Why must @naveentherocker do that before merge? Please explain.

**Comment by naveentherocker** [comment]
@niccokunzmann Please Review

**Comment by naveentherocker** [comment]
Both the extra two commits are verified @isuruAb @0x48piraj

**Comment by niccokunzmann** [comment]
@naveentherocker Thanks for squashing :) I will merge...

**Pull request merged**